guides/inventory.txt

```diff
@@ -0,0 +1,45 @@
+TOPIC
+    inventory
+
+DESCRIPTION
+    The inventory describes the targets that you run Bolt commands on, along
+    with any data and configuration for the targets. Targets in an inventory can
+    belong to one or more groups, allowing you to share data and configuration
+    across multiple targets and to specify multiple targets for your Bolt
+    commands without the need to list each target individually.
+
+    In most cases, the inventory is loaded from an inventory file in your Bolt
+    project. The inventory file is a YAML file named 'inventory.yaml'. Because
+    the inventory file is only loaded from a Bolt project, you must have an
+    existing project configuration file named 'bolt-project.yaml' alongside the
+    inventory file.
+
+EXAMPLE
+    The structure of an inventory file will vary depending on your project's
+    specific needs. The following example shows an inventory with targets
+    grouped by operating system:
+
+      --- 
+      # inventory.yaml
+      groups:
+        - name: linux
+          targets:
+            - nix-1.example.org
+            - nix-2.example.org
+          config:
+            ssh:
+              user: bolt
+              password: puppet
+        - name: windows
+          targets:
+            - win-1.example.org
+            - win-2.example.org
+          config:
+            transport: winrm
+            winrm:
+              user: bolt
+              password: puppet
+
+DOCUMENTATION
+    pup.pt/bolt-inventory
```

Comment:
`https://` would be helpful here as it's not totally obvious that these are URLs...

Reply:
They do kind of look like Bolt error kinds, don't they?